### ISSUE TITLE
Invalid filter: 'pjax' in profile page

### ISSUE THREAD

**Issue by alecsandruchirosca** [viewflow/django-material#25]
Dear all,

I followed the quickstart steps and accesing the admin->profile gives the folowing error:
Invalid filter: 'pjax'
1	
      
      {% extends 'material/frontend/base_site.html,material/frontend/base_pjax.html'|pjax:request %}
      

      
2	
3	{% block title %}{{ current_module.label }} - {{ block.super }}{% endblock %}

the same error on /accounts/profile/ page.

Please help me with this issue.

BR,
Alecs

**Comment by alecsandruchirosca** [comment]
solved by including easy_pjax in INSTALLED_APPS
I probably missed this setting - my mistake.

Thanks

**Comment by macagua** [comment]
:+1: thanks @alecsandruchirosca
